curbs/README.md

```diff
@@ -471,14 +473,14 @@ A Time Span is represented as a JSON object whose fields are as follows:
-| `increment_amount` | Integer | Optional | If specified, the rate for this space is rounded up to the nearest increment of this amount, specified in the same units as `per_hour`. |
-| `start_minutes` | Integer | Optional | The amount of time the vehicle must have already been present in the Curb Zone before this rate starts applying (_inclusive_, see [Range Boundaries](/general-information.md#range-boundaries)). If not specified, this rate starts when the vehicle arrives. |
-| `end_minutes` | Integer | Optional | The amount of time after which the rate stops applying (_exclusive_, see [Time Range](/general-information.md#time-range)). If not specified, this rate ends when the vehicle departs. |
+| Name                | Type    | Required/Optional | Description                                                                                                                                                                                                                                                   |
+|---------------------|---------|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `rate`              | Integer | Required          | The rate for this space in cents (or the smallest denomination of local currency) per `rate_unit`.                                                                                                                                                            |
+| `rate_unit`         | Enum    | Required          | The [Unit of Time Enum](/general-information.md#unit-of-time-enum) associated with the rate.                                                                                                                                                                  |
+| `increment_minutes` | Integer | Optional          | If specified, this is the smallest amount of time a user can pay for (e.g., if `increment_minutes` is `15`, a user can pay for 15, 30, 45, etc. minutes).                                                                                                     |
+| `increment_amount`  | Integer | Optional          | If specified, the rate for this space is rounded up to the nearest increment of this amount, specified in the same units as `per_hour`.                                                                                                                       |
+| `start_minutes`     | Integer | Optional          | The amount of time the vehicle must have already been present in the Curb Zone before this rate starts applying (_inclusive_, see [Range Boundaries](/general-information.md#range-boundaries)). If not specified, this rate starts when the vehicle arrives. |
```

Comment:
Do you know where 'Time Range' is supposed to go in the `end_minutes` field below @zanebclark ?

Reply:
I think it's suppose to be https://github.com/automotus/curb-data-specification/blob/zanebclark/max_stay_no_return_definition/general-information.md#range-boundaries like the others. I'll update.